app/cdash/xml_handlers/upload_handler.php

```diff
@@ -48,6 +48,8 @@ class UploadHandler extends AbstractHandler
     private $Base64TmpFileWriteHandle;
     private $Base64TmpFilename;
     private $Label;
+    private int $Timestamp;
```

Comment:
Would using Laravel's [Carbon](https://github.com/briannesbitt/Carbon) wrapper to manipulate these timestamps be realistic?  I know we're using legacy PHP date/time functions elsewhere in this file, so there's certainly an argument to be made for remaining consistent, but using Carbon will allow us to interact with datetimes more effectively once most things are switched to Eloquent.  Eloquent can auto-populate datetime fields in models with Carbon objects.